Dockerfile

```diff
@@ -1,6 +1,13 @@
 # The first instruction is what image we want to base our container on
```

Comment:
That's a lot of change, what was wrong with previous version?

Reply:
Nothing wrong in previous version, all production apps following similar Docker file manifest.